pkg/operator/ceph/cluster/osd/status_test.go

```diff
@@ -84,26 +86,35 @@ func mockNodeOrchestrationCompletion(c *Cluster, nodeName string, statusMapWatch
 				status = &OrchestrationStatus{
 					OSDs: []OSDInfo{
 						{
-							ID:      1,
+							ID:      osdID,
 							Cluster: "rook",
+							// set UUID to zero padded version of OSD ID (width 16)
+							UUID:      fmt.Sprintf("%016d", osdID),
```

Comment:
I thought OSDs had a random UUID (32 chars). Why pad the OSD ID to 16 chars? I know this is for a test so it must not matter, just curious.

Reply:
It was honestly just arbitrary. We don't need realistic-looking UUIDs for the test, but they need to be consistently set for OSDs, so doing this by OSD ID made sense.